ThirdPartyAdapters/unity/unity/src/main/java/com/google/ads/mediation/unity/UnityMediationAdapter.java

```diff
@@ -285,7 +299,68 @@ private static boolean isValidIds(String gameId, String placementId) {
 
             return false;
         }
-
         return true;
     }
+
+    @Override
+    public void onUnityAdsReady(String placementId) {
+        // Unity Ads is ready to show ads for the given placementId. Send Ad Loaded event if the
+        // adapter is currently loading ads.
+        if (placementId.equals(getPlacementId()) && mMediationAdLoadCallback != null) {
+            mMediationRewardedAdCallback = mMediationAdLoadCallback
+                    .onSuccess(UnityMediationAdapter.this);
+        }
```

Comment:
this is handled by load api callback

Reply:
So should mMediationRewardedAdCallback get set inside on onUnityAdsLoaded?